base/vault.yaml

```diff
@@ -0,0 +1,102 @@
+# This SA is used by vault to talk to the kube api and verify application's SAs
+# trying to get secrets
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: kube-authenticator
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: vault
+  labels:
+    app: vault
+  annotations:
+    fluentbit.io/exclude: "true"
```

Comment:
Fluentbit exclude shouldn't be in the base, but maybe we can make a note in the readme that first startup logs are sensitive

Reply:
I've moved it to the example with a comment explaining the rationale.